openassessment/assessment/models/base.py

```diff
@@ -552,6 +598,30 @@ def get_median_score(scores):
             )
         return median_score
 
+    @staticmethod
+    def get_mean_score(scores):
+        """Determine the median score in a list of scores
+
+        Determine the median value in the list.
+
+        Args:
+            scores (list): A list of int values. These int values
+                are reduced to a single value that represents the median.
+
+        Returns:
+            (int): The median score.
+
+        Examples:
+            >>> scores = 1, 2, 3, 4, 5]
+            >>> Assessment.get_median_score(scores)
+            3
+
+        """
+        total_criterion_scores = len(scores)
+        if total_criterion_scores == 0:
+            return 0
+        return int(math.ceil(sum(scores) / float(total_criterion_scores)))
```

Comment:
Should we always round down? or would it be better to round up the nearest integer?

Reply:
Yes, that was what was determined in the product proposal: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3891462181/Configurable+grading+strategy+for+peer+assessments